lang/en/logstore_xapi.php

```diff
@@ -111,6 +111,7 @@
 $string['logstorexapi:managehistoric'] = 'Manage historic data';
 
 // Info strings from xAPI errors
+$string['errorlogpagetitle'] = 'xAPI error log';
```

Comment:
I think we can remove this entry.
We should use `logstorexapierrorlog` instead.

Reply:
It was removed.